src/hooks/listHooks.js

```diff
@@ -218,6 +218,8 @@ export const useRenderItem = (query, navigation, options = {}) => {
     default: {
       /* eslint-disable complexity */
       renderItem = ({ item, index, section, target }) => {
+        const bottomDivider = item.bottomDivider ? true : calculateBottomDivider(index, section);
```

Comment:
we can do `const bottomDivider = item.bottomDivider ?? calculateBottomDivider(index, section)` like in the other places, right?

Reply:
Yes. To do this, I updated the `skipLastDivider` value in `DataListSection` to `false`.